src/Avalanche/InitialState.h

```diff
@@ -0,0 +1,40 @@
+// Copyright © 2017-2021 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+
+#pragma once
+
+#include "TransferableOutput.h"
+
+namespace TW::Avalanche {
+
+    class InitialState {
```

Comment:
what is `InitialState` for?

Reply:
`InitialState`s are used for creating an asset: https://docs.avax.network/build/references/avm-transaction-serialization#what-unsigned-create-asset-tx-contains